modules/stanford_paragraph_options/stanford_paragraph_options.module

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ * Contains stanford_paragraph_options.module.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use \Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_help().
+ */
+function stanford_paragraph_options_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the stanford_paragraph_options module.
+    case 'help.page.stanford_paragraph_options':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Limit the options of paragraph types for fields.') . '</p>';
+      return $output;
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function stanford_paragraph_options_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
the form_id is `node_complex_page_form` so we'd have to duplicate the function if we want to use it on other node forms. thoughts?